uk.co.compucorp.civicrm.hrcore/backstop_data/casper_scripts/ssp-leave-absences/staff/my-report-add-comment.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+var page = require('../../../page-objects/ssp-leave-absences');
+
+module.exports = function (casper) {
+  var myReport = page.init(casper).openMyReport();
```

Comment:
Based on my other comment, `openMyReport ` is not necessary as init would be calling `waitForReady` internally.